### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by hugoxiong5**
set up docker config and compose files; add placeholder image data; change bundle service url to point to deployed server

### CODE REVIEW

**Review by martinsung00 — APPROVED** [review]
Everything looks good, easy to understand, no unused variables. But on line 77 of client/PhotoModule.jsx -- did you need the console.log(e)? If not you might want to remove some development console.logs to keep it clean!

File: client/PhotoModule.jsx [hunk 1]
```diff
@@ -52,11 +64,21 @@ class PhotoModule extends React.Component {
...
+      const response = await fetch(`http://13.56.180.105/photos/${productId}/${styleId}`);
+      if (!response.ok) {
+        throw new Error('photo fetch request failed');
+      }
+      const data = await response.json();
+      this.setState({
+        photos: data,
+      });
+    } catch (e) {
+      console.log(e);
```
Comment (martinsung00) [inline client/PhotoModule.jsx]: you might want to switch it to throw (e) for production build rather than console logs

**Review by martinsung00 — CHANGES_REQUESTED** [review]

File: client/PhotoModule.jsx (hunk 1 above)
Comment (hugoxiong5) [reply client/PhotoModule.jsx]: I don't want to throw because that would stop the function. But I'll remove the console.log.

Comment (martinsung00) [reply client/PhotoModule.jsx]: Cool! Maybe try console.error to differentiate it from like development console logs!

**Review by martinsung00 — APPROVED** [review]

**Pull request merged**